docker-compose.yaml

```diff
@@ -98,8 +98,8 @@ services:
       <<: *spark-env
       SPARK_MODE: worker
       SPARK_MASTER_URL: spark://spark-node-master-prod:7077
-      SPARK_WORKER_MEMORY: 96g
-      SPARK_WORKER_CORES: 32
+      SPARK_WORKER_MEMORY: 96G
```

Comment:
**[Question, non-blocking]** Did you determine whether the capitalization matters here or not?

Reply:
Based on super quick tests, I don't believe it does. I just reverted it to the way it was before to be safe.